sdk/helper/keysutil/policy.go

```diff
@@ -692,16 +692,21 @@ func (p *Policy) Upgrade(ctx context.Context, storage logical.Storage, randReade
 	return nil
 }
 
-// DeriveKey is used to derive the encryption key that should be used depending
+// GetKey is used to derive the encryption key that should be used depending
 // on the policy. If derivation is disabled the raw key is used and no context
 // is required, otherwise the KDF mode is used with the context to derive the
 // proper key.
-func (p *Policy) DeriveKey(context []byte, ver, numBytes int) ([]byte, error) {
+func (p *Policy) GetKey(context []byte, ver, numBytes int) ([]byte, error) {
 	// Fast-path non-derived keys
 	if !p.Derived {
 		return p.Keys[strconv.Itoa(ver)].Key, nil
 	}
 
+	return p.DeriveKey(context, nil, ver, numBytes)
+}
+
+// DeriveKey is used to derive a symmetric key given a context and salt.
```

Comment:
Can you add a blurb to the comment about how this does not check the p.Derived flag?